File: VB/WebSite/ReportPreview.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Web.ASPxScheduler.Reporting

Partial Public Class ReportPreview
	Inherits System.Web.UI.UserControl
	Private controlAdapter_Renamed As ASPxSchedulerControlPrintAdapter

	Public Property ControlAdapter() As ASPxSchedulerControlPrintAdapter
		Get
			Return controlAdapter_Renamed
		End Get
		Set(ByVal value As ASPxSchedulerControlPrintAdapter)
			controlAdapter_Renamed = value
		End Set
	End Property

	Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
		Dim rpt As New XtraSchedulerReport1()
		Me.DataBind()
		rpt.SchedulerAdapter = ControlAdapter.SchedulerAdapter
		ReportViewer1.Report = rpt
	End Sub
End Class

File: CS/WebSite/ReportPreview.ascx.cs
using System;
using DevExpress.Web.ASPxScheduler.Reporting;

public partial class ReportPreview : System.Web.UI.UserControl {
    ASPxSchedulerControlPrintAdapter controlAdapter;

    public ASPxSchedulerControlPrintAdapter ControlAdapter { 
        get { return controlAdapter; } 
        set { controlAdapter = value; } }

    protected void Page_Load(object sender, EventArgs e) {
        XtraSchedulerReport1 rpt = new XtraSchedulerReport1();
        this.DataBind();
        rpt.SchedulerAdapter = ControlAdapter.SchedulerAdapter;
        ReportViewer1.Report = rpt;
    }
}
